api_blog/routes.py

```diff
@@ -1,5 +1,7 @@
-from .views import index
+from .views import index, sign_up, users_list
 
 
 def setup_routes(app):
     app.router.add_get("/", index)
+    app.router.add_get("/users_list/", users_list)
+    app.router.add_post("/sign_up/", sign_up)
```

Comment:
Do not use underscore(_) in your router. Replace it with a dash(-). And rename plz `users_list` to `users`